graphs/energy/nodes/energy/energy_hydrogen_steam_methane_reformer_ccs_must_run.ad

```diff
@@ -17,16 +17,16 @@
 - technical_lifetime = 25.0
 - variable_operation_and_maintenance_costs_for_ccs_per_full_load_hour = 0.0
 - variable_operation_and_maintenance_costs_per_full_load_hour = 0.0
-- wacc = 0.04
-- hydrogen.profile = flat
+- wacc = 0.07
+- hydrogen.group = flat
+- hydrogen.subtype = must_run
 - hydrogen.type = producer
 - network_gas.demand_carrier = natural_gas
 - network_gas.profile = flat
 - network_gas.type = consumer
-- forecasting_error = 0.0
 - free_co2_factor = 0.669
 - ccs_capture_rate = 0.669
-- land_use_per_unit = 0.03235581614498978
+- hydrogen_output_capacity = 300.0
 - takes_part_in_ets = 1.0
 
 ~ demand = 0.0
```

Comment:
@kndehaan shouldn't this hydrogen_output_capacity of SMR with CCS be lower due to the lower output efficiency compared to the SMR without CCS? Is this something you can look into?

Reply:
Regarding your new question about `hydrogen_output_capacity`, I see in ETEngine that there's a function to define  calculate this attribute if it is not already defined in the hydrogen node:
```
# Public: Returns the hydrogen output capacity of the node.
#
# If no value is assigned in the ETSource data, a capacity will attempt to be derived from the
# `typical_input_capacity`. Note that we do not call `input_capacity` and instead rely on the
# typical capacity, as `input_capacity` may itself call `hydrogen_output_capacity`.

      def hydrogen_output_capacity
        fetch(:hydrogen_output_capacity, false) do
          typical_input_capacity ? typical_input_capacity * hydrogen_output_conversion : 0.0
        end
      end
```

I'm not sure why we've defined `hydrogen_output_capacity` in some of the nodes, if this is really required for some reason or if it was first our intention to define this for all nodes and that later on we've decided to use a function to set this attribute. In the end, all hydrogen production nodes will obtain the `hydrogen_output_capacity` attribute. 

Do you think it's best to leave it as it is or to remove the defined `hydrogen_output_capacity` in the specific nodes and only use the defined function for all hydrogen production nodes?

Reply:
I think hydrogen_output_capacity can be set explicitly in case where it should not be determined by the output efficiency. For example for hydrogen storage we want to define input and output capacity independently.

I expect that we can remove it from the SMR and that would be my preference. In the future I would prefer to have all of our CCS technologies to receive a lower output capacity than their non-CCS counterpart instead of having a higher input capacity.

Reply:
I've removed the defined `hydrogen_output_capacity` attribute in the SMR, ATR and ammonia reforming nodes. https://github.com/quintel/etsource/pull/3021/commits/2bd22c7a91090f058ce1cec5efb595c9bf953c27